### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by MuhammadShameel**
I updated few styling in this.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,35 +1,46 @@
-// 1. Create An Array Of Students.
 let studentsOne = [
-  { name: "Hammad", age: 17 },
-  { name: "Abdul Rafay", age: 11 },
-  { name: "Huzaifa", age: 8 },
-  { name: "Laraib", age: 16 },
+  { Name: "Hammad", Age: 17, Class: "10th" },
```
Comment (Furqankhanzada) [inline script.js]: Why you changed then into capital letter? it always should be in small case.

File: script.js [hunk 2]
```diff
@@ -1,35 +1,46 @@
...
+      let row = document.createElement("tr");
+      for (let itemKey in item) {
+        let cell = document.createElement("td");
+        let textNode = document.createTextNode(itemKey);
+        cell.appendChild(textNode);
+        row.appendChild(cell);
+        table.appendChild(row);
+      }
+    }
+    let rowEl = document.createElement("tr");
```
Comment (Furqankhanzada) [inline script.js]: `thead` and `tboady` are missing have a look how the table should look https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead

File: script.js [hunk 3]
```diff
@@ -36,11 +37,22 @@ let rederTable = function (items) {
       cellEl.appendChild(textNode);
       rowEl.appendChild(cellEl);
     }
+    let deleteCell = document.createElement("td");
+    let deleteButton = document.createElement("button");
+    deleteButton.innerHTML = "Delete";
```
Comment (Furqankhanzada) [inline script.js]: can we use `. innerText` ? as we are just changing text?

File: script.js (hunk 1 above)
Comment (MuhammadShameel) [reply script.js]: I will make sure that it will never happen again and fix it.

File: script.js [hunk 4]
```diff
@@ -1,33 +1,57 @@
...
+  let thead = document.createElement("thead");
+  let tbody = document.createElement("tbody");
+
+  items.forEach((item, index) => {
+    if (index === 0) {
+      let row = document.createElement("tr");
+      for (let itemKey in item) {
+        let cell = document.createElement("td");
+        let textNode = document.createTextNode(itemKey);
+        cell.appendChild(textNode);
```
Comment (Furqankhanzada) [inline script.js]: instead of using `createTextNode` can we do  `cell.innerText == 'ANY_TEXT` ?

File: script.js [hunk 5]
```diff
@@ -1,33 +1,57 @@
...
+  { name: "Sir Bilal", age: 27, subjects: "English" },
+  { name: "Sir Areeb", age: 21, subjects: "Chemistry" },
+  { name: "Sir Nabeel", age: 28, subjects: "Physics" },
+  { name: "Sir Habeeb", age: 26, subjects: "Urdu" },
 ];
 
 // 2.Create a function named renderTable.
 let rederTable = function (items) {
   // 3. Create table dynamically .
   let container = document.querySelector("#table");
```
Comment (Furqankhanzada) [inline script.js]: Now we should take 2 arguments  `items` and 'elementId'

So we can decide where we will append the table.

```suggestion
let rederTable = function (items, elementId) {
  // 3. Create table dynamically .
  let container = document.querySelector(elementId);

```

**Pull request closed without merging**